ctapipe_io_lst/event_time.py

```diff
@@ -22,6 +22,7 @@
 
 
 CENTRAL_MODULE = 132
+INVALID_TIME = Time(0, format='mjd', scale='tai')
```

Comment:
Perhaps, better a more explicit  "ZER0_TIME"

Reply:
Zero is just a detail, invalid is the meaning. So I prefer invalid because it adds semantics.

Someting like this:

```
table['time'] == INVALID_TIME
```

Is more clear on the meaning than

```
table['time'] == ZERO_TIME
```

won't you agree?

Reply:
> table['time'] == ZERO_TIME
> ```
> 
> won't you agree?

Indeed it seems much more clear to me.

Reply:
if you like, I prefer to know, how it is set an invalid time, but it is a question of taste

Reply:
I just realized we also have it in ctapipe, so we should just import instead of define our own.